packages/client/src/components/LatestRatingMovies/LatestRatingMovies.jsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+import { apiURL } from '../../apiURL';
+import { Typography, Grid } from '@mui/material';
+import MovieCard from '../MovieCard/MovieCard';
+import MovieCard2 from '../MovieCard/MovieCard2';
+
+export default function LatestRatingMovies() {
+  const [movies, setMovies] = useState([]);
+  useEffect(() => {
+    async function fetchMovies() {
+      const response = await fetch(`${apiURL()}/movies`);
+      const examples = await response.json();
+      setMovies(examples);
+    }
+    fetchMovies();
+  }, []);
+  return (
+    <>
+      <Typography
+        sx={{
+          height: '2.76rem',
+          fontSize: '2.25rem',
+          fontWeight: 700,
+          color: '#FFFFFF',
+          textAlign: 'center',
+          marginBottom: '3rem',
+          marginTop: '3rem',
+          fontFamily: 'Inter',
+        }}
+      >
```

Comment:
As I mentioned in @maccrin PR, you guys might want to discuss in teams making some sort of component library internally in the project so you don't have to style everything all the time if the components repeat, like this header for example :) It is always easier later to change things as you have one source and maintain consitency, not to mention readability :)

Reply:
Yes, we should definitely do it. Thank you!